scwx-qt/source/scwx/qt/map/overlay_layer.cpp

```diff
@@ -22,7 +22,7 @@
 #include <QGuiApplication>
 #include <QMouseEvent>
 
-#if !(defined(_MSC_VER) || defined(__clang__))
```

Comment:
I think this include usage can go away completely. This is no longer used.

It was originally introduced here:
https://github.com/dpaulat/supercell-wx/commit/593010a